compilator/compilator_test.go

```diff
@@ -182,108 +182,134 @@ func TestCompilationRoleManifest(t *testing.T) {
 	}
 }
 
-func getContainerIDs(testRepository string) (string, error) {
-	val, err := exec.Command("docker", "ps", "-a", "--format={{.ID}}/{{.Image}}").CombinedOutput()
```

Comment:
Yes, it's a test.  I still couldn't stand having it around, though, because, really, we have this whole library thing…